news/add_expense-db_option_to_presentation_adder.rst

```diff
@@ -0,0 +1,30 @@
+**Added:**
+* Added --expense-db flag to a_presentation adder. If specified, it adds 
+  the expense item generated for the presentation to the specified database. 
+  Without a specification, by default the helper attempts to use the first 
+  database in the regolithrc.json file but will only do so if that database 
+  is non-public.
+
+* Added the --force option to the a_presentation adder. If specified, 
```

Comment:
this is too much detail for a changelog. no need to describe the force as it is just part of the expense-db functionality